tests/ctest/test_keyname.c

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file
+ *
+ * @brief Test suite for Libease functions accessing key name data.
+ *
+ * @copyright BSD License (see doc/COPYING or http://www.libelektra.org)
+ */
+
+#include <kdbease.h>
+
+#include "tests.h"
+
+#define ERROR_STRING_SIZE 1024
+
+static void test_relative (const char * expected, const char * result)
```

Comment:
> It might be useful to let test_relative be a macro: then the line information would be more useful.

Thanks for the advice. I just changed `test_relative` accordingly.